tasks/jscs.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+
+module.exports = function jscs(grunt) {
+    grunt.loadNpmTasks('grunt-jscs');
+
+    // Options
+    return {
+        options: {
+            config: '.jscsrc',
+            reporter: require('jscs-stylish').path
+        },
+        all: {
+            src: [
+                'engine.js',
```

Comment:
You'll probably want to remove this `engine.js` bit and add bits for checking `models`, `controllers` and `services` as well..